src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -181,22 +186,22 @@
 
     @ApiOperation(value = "Returns all documents' meta data for an offender by CRN")
     @ApiResponses(
-        value = {
-            @ApiResponse(code = 200, message = "OK", response = OffenderDocuments.class),
-            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
-            @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
-            @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
-            @ApiResponse(code = 404, message = "Not Found. For example if the CRN is not known.", response = ErrorResponse.class),
-            @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
-        })
+            value = {
+                    @ApiResponse(code = 200, message = "OK", response = OffenderDocuments.class),
```

Comment:
in general shouldn't document the 200 `OK` response as will be auto generated so therefore more likely to be correct